NOTICES.md

```diff
@@ -38,7 +32,8 @@ List of third-party dependencies grouped by their license type
 
 ### [MIT License](./licenses/mit_license.html):
 * Testcontainers :: Kafka ([org.testcontainers:kafka:1.14.3](https://testcontainers.org))
-* Testcontainers Core ([org.testcontainers:testcontainers:1.14.3](https://testcontainers.org))
+* Testcontainers :: Kafka ([org.testcontainers:kafka:1.15.3](https://testcontainers.org))
+* Testcontainers Core ([org.testcontainers:testcontainers:1.15.3](https://testcontainers.org))
```

Comment:
I've not been able to identify why there are 2 versions of this but it's not a blocker.

Reply:
When I run a build the 1.14.3 version gets removed from this file

Reply:
The kafka specialization (`KafkaContainer`) is distinct from our usage for redis (`GenericContainer`)